docs/source/troubleshooting.rst

```diff
@@ -18,10 +18,22 @@ Building
 Flashing
 --------
 
-* The board is not flashing.
+*Here are some of the tips if The board is not flashing..*
+
+* Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process.
+* Hold-down the **“BOOT”** button in your ESP32 board while unloading.
```

Comment:
Change `unloading` to `uploading/flashing`

Reply:
Okay, Done.